src/pkg/cloudclient/organizations/client.go

```diff
@@ -0,0 +1,65 @@
+package organizations
```

Comment:
@orishoshan the code around orgs & users was actually moved from the root `pkg` dir, and it was done using `git mv` (as you can see in the first commit in this PR), however it was then modified to support new commands, and github refuses to show it as "moved+edited". Apologies.